Formula/detekt.rb

```diff
@@ -0,0 +1,17 @@
+class Detekt < Formula
+  desc "Static code analysis for Kotlin"
+  homepage "https://github.com/arturbosch/detekt"
+  url "https://jcenter.bintray.com/io/gitlab/arturbosch/detekt/detekt-cli/1.1.1/detekt-cli-1.1.1-all.jar"
+  sha256 "8b17c25cd0e9c507773eae0afff97e89c4a1c502b1b1b91c8d55a62032326b3a"
+
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install "detekt-cli-1.1.1-all.jar"
+    bin.write_jar_script libexec/"detekt-cli-1.1.1-all.jar", "detekt"
+  end
+
+  test do
+    system bin/"detekt", "--help"
```

Comment:
Is it possible you could test actual functionality?

As per the [Formula Cookbook](https://docs.brew.sh/Formula-Cookbook):

> We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Reply:
I added fixed of commit.
It is the most simplified test, is this allright?

https://github.com/Homebrew/homebrew-core/pull/46981/commits/b65d87a6eadb8004e148d60267058eaa86c94506

Reply:
That's fine. It doesn't need to be complicated.